### PR TITLE
Pull stock BOOK data to the datasource

### PR DESCRIPTION
**Pull request by TitenkovEA**

### CODE REVIEW

File: datawell/iex.py [hunk 1]
```diff
@@ -31,6 +32,36 @@ def get_stocks(self):
...
+            ex = app.AppException(e, message)
+            raise ex
+
+    def get_all_books(self):
+        """
+        Gets all books for current stocks
+        :return: List of books
+        """
+        try:
+            return list(map(lambda stock: self.get_book(stock['symbol']), self.get_stocks()))
```
Comment (nikolai-riabinin-epam) [inline datawell/iex.py]: What do you think about using already populated Symbols instead of an additional API call?

Comment (petelind) [reply datawell/iex.py]: @nikolai-riabinin-epam has the radio - every API call costs you, both literally and figuratively :)

File: datawell/iex.py [hunk 2]
```diff
@@ -31,6 +32,36 @@ def get_stocks(self):
...
+        :param symbol: str with symbol from stock obj
+        :return: book as json obj
+        """
+        try:
+            uri = f'{app.BASE_API_URL}stock/{symbol}/book{app.API_TOKEN}'
+            book = self.load_from_iex(uri)
+            self.Books[symbol] = book
+            return book
+
+        except Exception as e:
```
[the remark this answers is not in the document]
Comment (TitenkovEA) [reply datawell/iex.py]: yeah, it was copy-pasted, to match to other methods in class. deleted, bc did'n find any expected error

File: datawell/iex.py (hunk 1 above)
Comment (TitenkovEA) [reply datawell/iex.py]: Usually in service classes we want to get the latest data from an API or database, so this is why I did it. But now, it seems that I understand the meaning of this class. So - fixed and refactored

**Pull request merged**